qm.te

```diff
@@ -30,11 +30,56 @@ unconfined_domain(ipc_t)
 
 qm_domain_template(qm)
 
+
+
+#########################################
+#
+# bluechi-agent inside QM
+#
+
+type qm_bluechi_agent_t;
```

Comment:
MHO this should go to 
template(`qm_domain_template',`
inside qm.if, 

But I am not selinux-expert

Reply:
As I wrote in https://github.com/containers/qm/issues/805#issuecomment-2850694999, we can further improve later by moving (most parts) to a dedicated iface in bluechi-selinux. I don't think that this should be part of the qm_domain_template.